apps/server/src/modules/fwu/uc/fwu.uc.ts

```diff
@@ -0,0 +1,44 @@
+import { Injectable } from '@nestjs/common';
+import { GetObjectCommand, S3Client } from '@aws-sdk/client-s3';
+import { Logger } from '@src/core/logger';
+import { Configuration } from '@hpi-schul-cloud/commons';
+
+@Injectable()
+export class FwuUc {
+	constructor(private logger: Logger) {
+		this.logger.setContext(FwuUc.name);
+	}
+
+	async get(path: string): Promise<Uint8Array> {
+		const client = new S3Client({
+			endpoint: Configuration.get('FWU_CONTENT__S3_ENDPOINT') as string,
+			credentials: {
+				accessKeyId: Configuration.get('FWU_CONTENT__S3_ACCESS_KEY') as string,
+				secretAccessKey: Configuration.get('FWU_CONTENT__S3_SECRET_KEY') as string,
+			},
+			region: Configuration.get('FWU_CONTENT__S3_REGION') as string,
+			tls: true,
+			forcePathStyle: true,
+		});
+		const request = new GetObjectCommand({
+			Bucket: Configuration.get('FWU_CONTENT__S3_BUCKET') as string,
+			Key: path.toString(),
+		});
+		const response = await client.send(request);
```

Comment:
Please create S3Client such as Provider. For example:
https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/apps/server/src/modules/files-storage/files-storage.module.ts#L31-L51

Reply:
The S3Clinet has been adapted accordingly.